style_exceptions/runtime_cpu_detection_allowlist.json

```diff
@@ -4,6 +4,7 @@
   "apache-arrow",
   "blis",
   "fftw",
+  "gismo",
```

Comment:
You don't use `ENV.runtime_cpu_detection`, so this isn't needed.

Reply:
Yes, I will change the formula accordingly.